.gitignore

```diff
@@ -4,6 +4,7 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
The `.venv/` line is correctly included to ignore virtual environment directories. However, ensure that this line is necessary for your project setup, as sometimes `.venv` might not be used. If you are using a different virtual environment directory, adjust accordingly.